### PR TITLE
Declarative routing for React.

### PR DESCRIPTION
**Pull request by hemanth**
@ryanflorence's initiative.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,10 +10,12 @@ In the area of open source, there's a lot of stuff happening around [ZEIT](https
 - [now-pipeline](https://github.com/bahmutov/now-pipeline) - Single command to deploy, run e2e tests and switch alias if tests pass.
 - [now-serve](https://github.com/zeit/now-serve) - *official* - Sub command for deploying static sites on now.
 
+
```
Comment (timneutkens) [inline README.md]: Might remove the extra whitespace line 👍

Comment (hemanth) [inline README.md]: 3oGRFkmoqoui9nzL2g

Comment (timneutkens) [inline README.md]: 😂  already removed it myself 😉

Comment (hemanth) [inline README.md]: 👌

**Comment by leo** [comment]
@hemanth @timneutkens How is this project related to NOW?

**Comment by hemanth** [comment]
It falls into Deployment Examples category

**Comment by leo** [comment]
But it's not specially related to ZEIT?

**Comment by CodeTheory** [comment]
This isn't so much a "Deployment Example" than a usage example. In which it doesn't really teach much apart from that `now` supports React Router.

I can't say I agree with this addition.

**Comment by leo** [comment]
@CodeTheory Can you revert it, please? 😊

**Pull request merged**